configs/defs/fallback.textproto

```diff
@@ -5491,140 +5701,1902 @@ benchmark_configs: {
+      "                  filter_regex: \"\\\\s*max_log_file_action(\\\\s|=).*\""
+      "                  expected_regex: \"max_log_file_action\\\\s*=\\\\s*keep_logs\""
+      "                }"
+      "              }"
+      "        }"
+      "    }"
+      "}"
+  }
+}
+benchmark_configs: {
+  id: "auditd-installed-fallback"
+  compliance_note: {
+    version: { cpe_uri: "fallback" version: "2.0.0" benchmark_document: "CIS Distribution Independent Linux" }
+    title: "Ensure auditd is installed"
+    description:
+      "auditd is the userspace component to the Linux Auditing System. "
+      "It's responsible for writing audit records to the disk"
+    rationale:
+      "The capturing of system events provides system administrators with "
+      "information to allow them to determine if unauthorized access to their system is occurring."
+    remediation:
+      "Run the following command to Install auditd \n"
+      "```# yum install audit audit-libs \n"
+      "# apt-get install auditd audispd-plugins \n"
+      "# zipper install audit audit-libs```"
+    cis_benchmark: {
+      profile_level: 2
+      severity: LOW
+    }
+    scan_instructions:
+      "generic:{check_alternatives:{"
+      "  file_checks:{"
+      "    files_to_check:{single_file:{path:\"/usr/bin/auditd\"}}"
+      "    existence:{should_exist: true}"
+      "  }"
+      "}"
+      "check_alternatives:{"
+      "  file_checks:{"
+      "    files_to_check:{single_file:{path:\"/usr/sbin/auditd\"}}"
+      "    existence:{should_exist: true}"
+      "  }"
+      "}"
+      "check_alternatives:{"
+      "  file_checks:{"
+      "    files_to_check:{single_file:{path:\"/bin/auditd\"}}"
+      "    existence:{should_exist: true}"
+      "  }"
+      "}"
+      "check_alternatives:{"
+      "  file_checks:{"
+      "    files_to_check:{single_file:{path:\"/sbin/auditd\"}}"
+      "    existence:{should_exist: true}"
+      "  }"
+      "}"
+      "}"
+  }
+}
+benchmark_configs: {
+  id: "auditd-service-enabled-fallback"
+  compliance_note: {
+    version: { cpe_uri: "fallback" version: "2.0.0" benchmark_document: "CIS Distribution Independent Linux" }
+    title: "Ensure auditd service is enabled"
+    description:
+      "Turn on the auditd daemon to record system events."
+    rationale:
+      "The capturing of system events provides system administrators with "
+      "information to allow them to determine if unauthorized access to their system is occurring."
+    remediation:
+      "Run one of the following commands to enable auditd : \n"
+      "```# chkconfig auditd on \n"
+      "# systemctl enable auditd \n"
+      "# update-rc.d auditd enable \n"
+      "Notes: Additional methods of enabling a service exist. "
+      "Consult your distribution documentation for appropriate methods."
+    cis_benchmark: {
+      profile_level: 2
+      severity: LOW
+    }
+    scan_instructions:
+      "generic:{"
+      "  check_alternatives:{"
+      "    file_checks:{"
```

Comment:
Add check_alternative for /etc/systemd/system/ not existing

Reply:
This check_alternative is still missing - if I run Localtoast chrooted to an empty dir I get two failed benchmarks, `selinux-policy-configured-fallback` and `auditd-service-enabled-fallback`